modules/pixi/PixiLayerOsm.js

```diff
@@ -292,8 +292,12 @@ export class PixiLayerOsm extends AbstractLayer {
         feature.container.zIndex = -area;      // sort by area descending (small things above big things)
 
         const geojson = geojsonRewind(entity.asGeoJSON(graph), true);
-        const geometry = (geojson.type === 'Polygon') ? [geojson.coordinates]
+
+        // Even though we're rendering polygons, we also want to render any polys while they're being drawn.
```

Comment:
Should we instead send any two-vertex areas (i.e. ones that are still in the middle of being drawn) to the line renderer? As a treat?